pom.xml

```diff
@@ -18,12 +18,12 @@
         <!-- lookup parent from repository -->
     </parent>
 
-	<scm>
-		<url>https://github.com/spring-cloud/spring-cloud-consul</url>
-		<connection>scm:git:git://github.com/spring-cloud/spring-cloud-consul.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/spring-cloud/spring-cloud-consul.git</developerConnection>
-		<tag>HEAD</tag>
-	</scm>
+    <scm>
```

Comment:
Try not to reformat.  We have code conventions here: https://github.com/spring-cloud/spring-cloud-netflix#code-conventions-and-housekeeping-1  That will get added to this project soon.